docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -374,8 +374,8 @@ Json Parameters:
     -   **Binds** – A list of volume bindings for this container. Each volume binding is a string in one of these forms:
            + `host_path:container_path` to bind-mount a host path into the container
            + `host_path:container_path:ro` to make the bind-mount read-only inside the container.
-           + `volume_name:container_path` to bind-mount a volume managed by a volume plugin into the container.
-           + `volume_name:container_path:ro` to make the bind mount read-only inside the container.
+           + `volume_name:container_path[:nocopy]` to bind-mount a volume managed by a volume plugin into the container. Volume contents are initialized with data from `container-dest`, provided that the volume is empty and that `container-dest` is not declared as `VOLUME`. Such an initialization can be avoided by using `nocopy` option.
+           + `volume_name:container_path:ro[,nocopy]` to make the bind mount read-only inside the container. Volume contents are initialized with data from `container-dest`, provided that the volume is empty and that `container-dest` is not declared as `VOLUME`. Such an initialization can be avoided by using `nocopy` option.
```

Comment:
AFAIK, if container-dest is declared as VOLUME, data are not copied. I 'll check again